tests/python/relay/test_pass_alpha_equal.py

```diff
@@ -437,16 +423,13 @@ def test_op_alpha_equal():
     test_tensor_type_alpha_equal()
     test_incomplete_type_alpha_equal()
     test_constant_alpha_equal()
-    test_type_param_alpha_equal()
```

Comment:
Why have `test_type_param_alpha_equal()` and `test_var_alpha_equal()` been eliminated from the main method? Those test cases don't appear to have been eliminated.

Reply:
was removed because TypeParam  nolonger exists